docs/user/examples/water_tank_FI/README.md

```diff
@@ -0,0 +1,67 @@
+# Water Tank Fault Injection
+
+## Overview
+The co-simulation is composed of a water-tank, and a controller, which aims to maintain the level in the water-tank between ```1``` and ```2```. 
+More details on this case-study can be found [here](https://github.com/INTO-CPS-Association/example-single_watertank).
+
+To run an experiment with fault-injection, we inject a fault in the input of the tank (control signal from controller to open or close), such that between time 12 and 20, this signal is always closed, irrespective of the actual level in the tank.
+
+# Example Diagram
+
+![Water Tank System](watertank.png)
+
+# Example Structure
+
+![Water Tank Structure](watertank_structure.png)
+
+## Configuration of assets
+
+This example uses two models and one tool. The specific assets used are:
+
+| Asset Type | Names of Assets | Visibility | Reuse in Other Examples |
+|:---|:---|:---|:---|
```

Comment:
The two models used are:

watertankcontroller-c.fmu",
singlewatertank-20sim.fmu"